android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -145,6 +144,9 @@ public synchronized void onPrepared(MediaPlayer mp) {
 
       @Override
       public synchronized boolean onError(MediaPlayer mp, int what, int extra) {
+        if (module.playerPool.get(key) != null) {
```

Comment:
So, I'm wondering if the `key` variable safe even after return.

Reply:
Wow... I am not sure, but I guess, `key` was set to `playerPool`. Thus, it maybe safe...?
I will check later!